speechbrain/utils/torch_audio_backend.py

```diff
@@ -26,31 +28,32 @@ def try_parse_torchaudio_major_version() -> Optional[int]:
 
     version_split = torchaudio.__version__.split(".")
 
-    # expect in format x.y.zwhatever; we care only about x # cspell:ignore zwhatever
+    # expect in format x.y.z whatever; we care only about x # cspell:ignore zwhatever
```

Comment:
Good fix, I don't think we need the cspell directive anymore
```suggestion
    # expect in format x.y.z whatever; we care only about x
```